internal/core/data/router.go

```diff
@@ -1106,7 +1106,7 @@ func valueDescriptorHandler(w http.ResponseWriter, r *http.Request) {
 				w,
 				err,
 				[]errorconcept.ErrorConceptType{
-					errorconcept.Common.JsonDecoding,
+					errorconcept.CoreData.JsonDecoding,
```

Comment:
Any reason why errorConept.Common.JsonDecoding can't be used here?

This comment ties into my comment in coredata.go